### ISSUE TITLE
Missing '?' and '&' characters in query string of output

### ISSUE THREAD

**Issue by joshbader** [mckeever02/mailtolink#2]
Missing '?' and '&' characters in query string of output prior clicking the copy button.

**Comment by mckeever02** [comment]
Yeah the formatting is ran when the copy button is clicked. Seemed like an easier solution than listening to key events.

**Comment by joshbader** [comment]
You could always try throttling or debouncing the keyup event and then run the formate function.

**Comment by insign** [comment]
I identified some bug in my tests: mailto:a@helio.me?cc=dsdsd@gmail.com&?subject=Teste&body=asdasdsadasdasda

**Comment by mckeever02** [comment]
@insign Do you remember what order you entered in the form to create the error? I'm having trouble reproducing.

**Comment by insign** [comment]
Let's say that was email, subject, body then cc

**Comment by mckeever02** [comment]
@insign Sorry, long delay.  Could you try clearing your browser cache/a different browser? I've tried to replicate this and can't.

**Comment by mhsattarian** [comment]
@mckeever02 
if the form is filled in order below, there is an extra `?` before body like : `...&?body=...` in the query!

```
email -> body -> subject
```

**Comment by uncenter** [comment]
I've forked the project at https://github.com/uncenter/mailtolink and fixed this issue, FYI.
